_includes/v3.6/release-notes/v3.6.0-release-notes.md

```diff
@@ -0,0 +1,38 @@
+09 Mar 2019
+
+#### Calico IPAM for Kubernetes API datastore
+
+You now have full feature parity for Calico IP address management (IPAM) irrespective of the datastore you are using with your Calico deployment. This enables all of Calico’s
```

Comment:
I think this could be tightened up a bit with bullet points:

> When using the Kubernetes API Datastore, Calico IPAM now has full feature parity with the etcd datastore.  Including support for:
> - multiple IP pools
> - per-pod, per-namespace and per-node IP pool specification.